book/src/advanced-usage/sdk/evm-proof.md

```diff
@@ -46,37 +50,23 @@ The `StdIn` struct allows you to format any serializable type into a VM-readable
 > **Generating CLI Bytes**  
 > To get the VM byte representation of a serializable struct `data` (i.e. for use in the CLI), you can print out the result of `openvm::serde::to_vec(data).unwrap()` in a Rust host program.
 
-## Generating Proofs
+## Generating App Proofs
 
 After building and transpiling a program, you can then generate a proof. To do so, you need to commit your `VmExe`, generate an `AppProvingKey`, format your input into `StdIn`, and then generate a proof.
 
 ```rust,no_run,noplayground
-{{ #include ../../../crates/sdk/examples/sdk.rs:proof_generation }}
-```
-
-## Verifying Proofs
-
-After generating a proof, you can verify it. To do so, you need your verifying key (which you can get from your `AppProvingKey`) and the output of your `generate_app_proof` call.
-
-```rust,no_run,noplayground
-{{ #include ../../../crates/sdk/examples/sdk.rs:verification }}
+{{ #include ../../../../crates/sdk/examples/sdk_app.rs:proof_generation }}
 ```
 
 ## End-to-end EVM Proof Generation and Verification
 
 Generating and verifying an EVM proof is an extension of the above process.
```

Comment:
Edit this line to emphasize that `generate_app_proof` does not need to be called separately